test/cmdlineTests/function_debug_info/input.sol

```diff
@@ -0,0 +1,6 @@
+contract C {
```

Comment:
I'd add license and version pragma to silence the warnings. They're just noise here.
```suggestion
// SPDX-License-Identifier: GPL-3.0
pragma solidity >=0.0;

contract C {
```

Same in the other test.